src/nn.h

```diff
@@ -43,12 +44,14 @@ typedef unsigned __int64 NN_UINT64;
 #define NN_UINT64 uint64_t
 #endif
 
-/*  Handle DSO symbol visibility                                             */
+/*  Handle DSO symbol visibility. */
```

Comment:
I filed a bug to clean this up too.  NN_EXPORT should be extern on all non-windows systems, we can clean this crap up.